src/commands/link.ts

```diff
@@ -0,0 +1,53 @@
+import { logger } from "../logger";
+import {
+    SlashCommandBuilder,
+    SlashCommandStringOption,
+} from "@discordjs/builders";
+import { CommandInteraction, CacheType, AutocompleteInteraction } from "discord.js";
+import { CommandType } from "../types";
+import { FilterLinkByName, FindLinkByName, GetAllLinks, Link } from "../helpers/linkQueries";
+
+const linkModule: CommandType = {
+    data: new SlashCommandBuilder()
+        .setName("link")
+        .setDescription("Which link do you want to send")
+        .addStringOption((option: SlashCommandStringOption) =>
+            option
+                .setName("link")
+                .setDescription("Select a link")
+                .setRequired(true)
+                .setAutocomplete(true)
+        ),
+    execute: async (interaction: CommandInteraction<CacheType>) => {
+        try {
+            const choice = interaction.options.getString("link", true);
+            let res = FindLinkByName(choice);
+            if (res) {
+                await interaction.reply({
+                    content: `Your request for ${res.shorten_link}: ${res.url}`
```

Comment:
I think its probably better to just return the actual url instead of this preformatted message